gammapy/maps/wcs/geom.py

```diff
@@ -316,36 +314,40 @@ def create(
 
         Parameters
         ----------
-        npix : int or tuple or list
+        npix : int or tuple or list, optional
             Width of the map in pixels. A tuple will be interpreted as
             parameters for longitude and latitude axes.  For maps with
             non-spatial dimensions, list input can be used to define a
             different map width in each image plane.  This option
```

Comment:
By the way, there are a lot of places in `maps` where there is two space instead of one like this one. It is not strictly bothering because the doc don't render this additional space. But it gets me when I read the docstring ...

```suggestion
            different map width in each image plane. This option
```